auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -467,10 +469,11 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
 
   async showAddSuccessModalNR () {
     this.$refs.addNRDialog.close()
-    this.dialogTitle = 'Name Request Added'
-    this.dialogText = 'You have successfully added a name request'
     await this.syncBusinesses()
-    this.$refs.successDialog.open()
+    this.highlightRowIndex = 1
+    setTimeout(() => {
+      this.highlightRowIndex = NaN
+    }, 4000)
```

Comment:
So, THIS component keeps track of which row to highlight and for how long. (How to style it is AffiliatedEntityTable's job.) These are not the VDataTable's concerns.